tests/unit/storageProviders/fs/fsSourcesStorageProvider.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { promises } from 'fs';
+import fs from 'node:fs';
 import path from 'path';
```

Comment:
please rename all "path" imports to "node:path"

Reply:
done

Reply:
done